content/outcomes/application-observability/tsm-slo-based-measurement-and-monitoring-apps.md

```diff
@@ -0,0 +1,85 @@
+---
+date: '2021-07-13'
+lastmod: '2021-07-13'
+layout: single
+team:
+  - Mark Schweighardt
+  - Niran Even-Chen
+  - Andrew Huffman
+title: "Tanzu Service Mesh: SLO-based Measurement & Monitoring of Applications"
+weight: 103
+---
+
+## Overview
+
+TSM Service level objectives (SLOs) provide a formalized way for Application Operators, Application Owners, and Application Developers to describe, measure, and monitor the performance, quality, and reliability of microservice applications in an automated way.  
```

Comment:
- Capitalize _Service Level Objectives_
- I might describe what an SLO is in general first, and then say something about how TSM helps. Referring to a "TSM SLO" is a bit confusing.

Reply:
took a crack at fixing this